boards/shared/Inc/App/App_SharedSetPeriodicCanSignals.h

```diff
@@ -34,3 +34,19 @@
                                                                                \
         can_signal_setter(can_tx, value);                                      \
     }
+
```

Comment:
I'll remove this macro toms

Reply:
We determined that BinaryStatus is a more common use case. Keeping the macro for now.